src/fu-tool.c

```diff
@@ -1853,6 +1855,17 @@ fu_util_get_report_metadata(FuUtilPrivate *priv, gchar **values, GError **error)
 			fu_util_report_metadata_to_string(metadata_post, 3, str);
 		}
 	}
+
+	/* plugin metadata */
+	plugins = fu_engine_get_plugins(priv->engine);
```

Comment:
If you're getting all the plugins metadata, I wonder if you should also get all the enabled plugins for the report too.

Reply:
Ya, but not for this PR. Thx.